cardano-cli/src/Cardano/CLI/EraBased/Transaction/Run.hs

```diff
@@ -1789,12 +1775,15 @@ runTransactionCalculatePlutusScriptCostCmd
           )
         $ encodePretty scriptCostOutput
 
+    castUtxoFileEra :: File (UTxO era1) In -> File (UTxO era2) In
```

Comment:
We really shouldn't need to be casting eras and this an be a potential source of ambiguous errors if we cast from an older to a newer era.